app/views/adopters/steps/_your_home.html.erb

```diff
@@ -1,62 +1,92 @@
-  <div class="control-group">
+<div class="control-group">
     <%= af.label :spouse_name, "Spouse/Partner's name", :class => "control-label" %>
     <div class="controls">
-      <%= af.text_field :spouse_name %>
+        <%= af.text_field :spouse_name %>
     </div>
-  </div>
-  <div class="control-group">
-    <%= af.label :other_household_names, "Other Household Members", :class => "control-label" %>
-    <div class="controls">
-      <%= af.text_field :other_household_names %>
-      <span class="help-inline">Please include names and ages.</span>
+</div>
+<div class="control-group">
+
+    <div class="control-group">
+        <%= af.label :how_did_you_hear, "How did you hear about us?", :class => "control-label" %>
+        <div class="controls">
+            <%= af.text_field :how_did_you_hear %>
+            <span class="help-inline">Google, Craigslist, Petfinder, etc.</span>
+        </div>
     </div>
-  </div>
-  <div class="control-group">
-    <%= af.label :how_did_you_hear, "How did you hear about us?", :class => "control-label" %>
-    <div class="controls">
-      <%= af.text_field :how_did_you_hear %>
-      <span class="help-inline">Google, Craigslist, Petfinder, etc.</span>
+    <div class="control-group">
+        <%= af.label :ready_to_adopt_dt, "How soon do you want to adopt?", :class => "control-label" %>
+        <div class="controls">
+            <%= af.text_field :ready_to_adopt_dt, :class => "required" %>
+            <span class="help-inline">Enter as YYYY-MM-DD</span>
+        </div>
     </div>
-  </div>
-  <div class="control-group">
-    <%= af.label :ready_to_adopt_dt, "How soon do you want to adopt?", :class => "control-label" %>
-    <div class="controls">
-      <%= af.text_field :ready_to_adopt_dt, :class => "required" %>
-      <span class="help-inline">Enter as YYYY-MM-DD</span>
+    <div class="control-group">
+        <%= af.label :is_ofage, "Are you at least 21 years old?", :class => "control-label" %>
+        <div class="controls">
+            <label>
+                <%= af.radio_button :is_ofage, true, :class => "required" %>
+                <span>Yes</span>
+            </label>
+            <label>
+                <%= af.radio_button :is_ofage, false, :class => "required" %>
+                <span>No</span>
+            </label>
+        </div>
     </div>
-  </div>
-  <div class="control-group">
-    <%= af.label :is_ofage, "Are you at least 21 years old?", :class => "control-label" %>
-    <div class="controls">
-      <label>
-        <%= af.radio_button :is_ofage, true, :class => "required" %>
-        <span>Yes</span>
-      </label>
-      <label>
-        <%= af.radio_button :is_ofage, false, :class => "required" %>
-        <span>No</span>
-      </label>
+    <div class="control-group">
+        <%= af.label :other_household_names, "Is there anyone under 18 years old?", :class => "control-label" %>
```

Comment:
```other_household_names``` already exists as in-use value that contains names and ages as a string.  So it isn't really suitable to be repurposed as a boolean.  

A new boolean column in the adoption application form would be required since this is a new value.

Reply:
Just to clarify, I would want to add a boolean column to which model? Is it the one that is adoption apps?

Reply:
Yes it should get added to ```adoption_app``